omod/src/main/resources/config.xml

```diff
@@ -12,10 +12,14 @@
 		${project.parent.description}
 	</description>
 	<updateURL>https://modules.openmrs.org/modules/download/${project.parent.artifactId}/update.rdf</updateURL>
-	<require_version>${openMRSVersion}</require_version>

```

Comment:
What is the reason behind hard coding this instead of the original variable?

Reply:
ok let me work on it sebo